docs/sources/breaking-changes/breaking-changes-v10-0.md

```diff
@@ -0,0 +1,283 @@
+---
+description: Breaking changes for Grafana v10.0
+keywords:
+  - grafana
+  - breaking changes
+  - documentation
+  - '10.0'
+  - release notes
+title: Breaking changes in Grafana v10.0
+weight: -1
+---
+
+# Breaking changes in Grafana v10.0
+
+Following are breaking changes that you should be aware of when upgrading to Grafana v10.0. 
+
+For our purposes, a breaking change is any change that requires users or operators to do something. This includes:
+
+- Changes in one part of the system that could cause other components to fail
+- Deprecations or removal of a feature
+- Changes to an API that could break automation
+- Changes that affect some plugins or functions of Grafana
+- Migrations that can’t be rolled back
+
+For each change, the provided information:
+
+- Helps you determine if you’re affected
+- Describes the change or relevant background information
+- Guides you in how to mitigate for the change or migrate
+- Provides more learning resources
+
+For release highlights and deprecations, refer to our [v10.0 What’s new]({{< relref "../whatsnew/whats-new-in-v10.0/index.md" >}}). For the specific steps we recommend when you upgrade to v10.0, check out our [Upgrade guide]({{< relref "../upgrade-guide/upgrade-v10.0/index.md" >}}).
+
+
+<!-- Template below
+## Feature
+
+
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## General breaking changes
+
+### Angular is deprecated and turned off by default for new Grafana Cloud stacks
+
+#### You are affected if
+
+You create a new Grafana Cloud stack and intend to use any panel or data source plugins written using the Angular.js JavaScript framework. See the list of Angular plugins here.
```

Comment:
The list is still in review so we may need to update this page when it is live - https://github.com/grafana/grafana/pull/68096